src/httpserver.cpp

```diff
@@ -300,6 +300,22 @@ static void connection_close_cb(evhttp_connection* conn, void *arg)
     }
 }
 
+#if LIBEVENT_VERSION_NUMBER >= 0x02020001
```

Comment:
awesome that you're using the callback - one way to unambiguously detect the presence of the new feature would be to add a new autoconf test maybe using AC_LINK_IFELSE for evhttp_set_newreqcb